public/main.css

```diff
@@ -160,31 +174,61 @@ tbody {
 }
 
 /**   upload-progress   **/
-#progress-bar {
-  width: 300px;
-  height: 5px;
-  background: linear-gradient(
-    90deg,
-    #FD9800,
-    #D73000 var(--progress),
-    white var(--progress)
-  );
-  border: 0.5px solid;
-  border-radius: 5px;
+.progress-bar {
+  margin-top: 3px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
 }
 
-/**   share-link    **/
-.share-window {
+.percentage {
+  position: absolute;
+}
+
+.percent-number {
+  font-size: 43.2px;
+  letter-spacing: -0.78px;
+  line-height: 58px;
+  font-family: Segoe UI, 'SF Pro Text', sans-serif;
+}
+
+.percent-sign {
+  font-size: 28.8px;
+  color: rgb(104, 104, 104);
+  letter-spacing: -0.78px;
+  font-family: Segoe UI, 'SF Pro Text', sans-serif;
+}
+
+.upload {
   margin: 0 auto;
-  width: 470px;
-  height: 250px;
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+.progress-text {
+  font-family: 'SF Pro Text', sans-serif;
```

Comment:
same here, some of these rules can pass from `.upload` to both `#cancel-upload` and `progress-text`
it might be worth setting `font-family: 'SF Pro Text';` on the body, then only making rules for the ones that are different?